README.md

```diff
@@ -6,24 +6,24 @@ develop [![Continuous Build Develop Branch](http://build.scenarioo.org/jenkins/j
 
 ## Automated Documentation of Applications with UI Tests
 
-Scenarioo is a tool to leverage the power of your UI Tests and to make them available as a documentation of your software system to all people involved in the design, development, test, operation and management of your software system.
+Scenarioo is a tool to leverage the power of your UI tests and to make them available as a documentation of your software system to all people involved in the design, development, test, operation and management of your software system.
 
-### Webpage and Documentation
+### Webpage & Documentation
```

Comment:
take care - changing titles always means that links to here could be broken. Here in this case I think it is not a problem, but should be checked for every title change.